assets/sass/widgets/_widgets.scss

```diff
@@ -45,6 +45,12 @@
 	&__body,
 	&__footer {
 
+		& .googlesitekit-table,
+		& > .mdc-layout-grid {
+			margin: 0;
+			padding: 0;
+		}
+
```

Comment:
This change should also be reverted, as it visually breaks the All Traffic widget in the dashboard. It shouldn't be needed in the first place if we use the approach from #3015 for the widget header. Either you can copy the components from there to here and go with the same approach right away (shouldn't cause a merge conflict if you use the exact same code), or you can add a temporary CSS rule that only affects the specific widget here.

Reply:
Yep, copied those components and reverted my css chagnes.